### PR TITLE
update name of CH1 & CH3

### PR DESCRIPTION
**Pull request by mucheinz**

### CODE REVIEW

File: ebusd-2.1.x/en/vaillant/15.uih.csv [hunk 1]
```diff
@@ -48,7 +48,9 @@ r;wi,,EhpHeatBufferAvailable,EhpHeatBufferAvailable,,,,"5C00",,,onoff,,,heat buf
 [SW>=250]r,,SysWithCoolingSupport,SysWithCoolingSupport,,,,"9B00",,,onoff,,,
 [SW>=362]r,,reserve1,reserve 1,,,,"9D00",,,UCH,,,for future use
 # ##### HC Names and Tel No #####,,,,,,,,,,,,,
+r;w,,HcName1,aaucHcNames CH1,,,,"4100",,,STR:11,,,name of CH1
```
Comment (john30) [inline ebusd-2.1.x/en/vaillant/15.uih.csv:51]: is there really a heating circuit 1? usually this is some internal stuff and I doubt that the name for this is anywhere visible on the controller

Comment (mucheinz) [reply ebusd-2.1.x/en/vaillant/15.uih.csv:51]: Hello John, yes there is - see results from the reads of the corresponding calls:
read -c uih HcName1
**HK1**       
read -c uih HcName2
**HK2**       
read -c uih HcName3
**HK3**       
read -c uih HcName4
**HK4**       
read -c uih HcName16
**Speicher**

Comment (john30) [reply ebusd-2.1.x/en/vaillant/15.uih.csv:51]: sure it is readable on the bus, but can you see the name of HK1 anywhere in the menu of the controller?

Comment (mucheinz) [reply ebusd-2.1.x/en/vaillant/15.uih.csv]: if you feel it is more correct, then leave CH1 out, but add CH3

**Review by mucheinz — COMMENTED** [review]
removed CH1

File: ebusd-2.1.x/en/vaillant/15.uih.csv [hunk 2]
```diff
@@ -49,6 +49,7 @@ r;wi,,EhpHeatBufferAvailable,EhpHeatBufferAvailable,,,,"5C00",,,onoff,,,heat buf
 [SW>=362]r,,reserve1,reserve 1,,,,"9D00",,,UCH,,,for future use
 # ##### HC Names and Tel No #####,,,,,,,,,,,,,
 r;w,,HcName2,aaucHcNames CH2,,,,"4108",,,STR:11,,,name of CH2
+r;w,,HcName3,aaucHcNames CH3,,,,"4110",,,STR:11,,,name of CH3
```
Comment (john30) [inline ebusd-2.1.x/en/vaillant/15.uih.csv:52]: please add it on the german version as well

**Review by john30 — CHANGES_REQUESTED** [review]

File: ebusd-2.1.x/en/vaillant/15.uih.csv (hunk 2 above)
Comment (mucheinz) [reply ebusd-2.1.x/en/vaillant/15.uih.csv:52]: added to ebusd-2.1.x/de/vaillant/15.uih.csv

**Pull request merged**